tests/fixtures/sample_lib/data_model/sample.py

```diff
@@ -29,6 +30,7 @@ class OtherOutputType(DataObjectBase):
     """A simple return type for the `other_task` task"""
 
     farewell: str
+    producer_id: ProducerId
```

Comment:
Ideally I want to be able to test this without changing the actual task definition. Mocks maybe?